cypress/e2e/shared/nav.test.ts

```diff
@@ -158,6 +163,14 @@ describe('navigation', () => {
       'templates',
       'labels',
     ].forEach(x => {
+      if (x === 'history') {
+        cy.getByTestID('nav-item-alerting').click()
+      } else if (x === 'variables' || x === 'templates' || x === 'labels') {
+        cy.getByTestID('nav-item-settings').click()
+      } else {
+        cy.getByTestID('nav-item-load-data').click()
+      }
+
```

Comment:
Because the way new navigation works, we have to open up tabs before we can click on the sub menu items. Before clockface 3, the submenu items were open by default.